src/test/java/com/salesforce/dynamodbv2/testsupport/ArgumentBuilder.java

```diff
@@ -111,18 +114,7 @@ public TestArgumentSupplier(AmazonDynamoDB rootAmazonDynamoDb) {
          * byAccount
          */
         AmazonDynamoDB byAccount = MtAmazonDynamoDbByAccount.accountMapperBuilder()
-            .withAccountMapper(new MtAccountMapper() {
-                @Override
-                public AmazonDynamoDB getAmazonDynamoDb(MtAmazonDynamoDbContextProvider mtContext) {
-                    return wrapWithLogger(getServers().get(
-                        Integer.valueOf(mtContext.getContext().split("-")[1]) % ORGS_PER_TEST).start());
-                }
-
-                @Override
-                public void shutdown() {
-                    getServers().forEach(LocalDynamoDbServer::stop);
-                }
-            })
+            .withAccountMapper(new DynamicAccountMtMapper())
             .withContext(MT_CONTEXT).build();
 
         /*
```

Comment:
perhaps leave a quick comment that 'endsWith("3")' is an example of a zany, arbitrary rule